src/net.h

```diff
@@ -1099,7 +1104,10 @@ class CConnman
     AddrMan& addrman;
     std::deque<std::string> m_addr_fetches GUARDED_BY(m_addr_fetches_mutex);
     Mutex m_addr_fetches_mutex;
-    std::vector<std::string> m_added_nodes GUARDED_BY(m_added_nodes_mutex);
+
+    // connection string and whether to use v2 p2p
+    std::vector<std::pair<std::string, bool>> m_added_nodes GUARDED_BY(m_added_nodes_mutex);
```

Comment:
Personally think adding a small struct with two vars is superior to this std::pair/tie stuff where you have to look up/remember what the components are.

Reply:
Done in #24545.